packages/astro/actions-module.template.mjs

```diff
@@ -0,0 +1,49 @@
+import { ActionError, callSafely } from 'astro:actions';
+
+function toActionProxy(actionCallback = {}, aggregatedPath = '/_actions/') {
+	return new Proxy(actionCallback, {
+		get(target, objKey) {
+			const path = aggregatedPath + objKey.toString();
+			if (objKey in target) {
+				return target[objKey];
+			}
+			async function action(param) {
```

Comment:
can you lift this function out of the scope? Preferably to the top-level? So that way it's not recreated (consuming memory) on every invocation.

Reply:
Good point, can do!